scripts/visualizer/src/server/index.js

```diff
@@ -5,9 +5,11 @@
  */
 const Linker = require('ducks/billsMatching/Linker/Linker').default
 const { cozyClient } = require('cozy-konnector-libs')
-const { fetchAll } = require('ducks/billsMatching/utils')
+const { Bill } = require('models')
 const path = require('path')
 
+Bill.registerClient(cozyClient)
```

Comment:
You can use Document here, it will be the same but it will be less confusing. Here another class extending Document will also be registered which might be surprising, it is less surprising if you use Document since it is the mother class.